scripts/gulp/tasks/gulpWebpack.ts

```diff
@@ -22,14 +22,16 @@ type RunWebpackCfg = {
 
 export async function runWebpack (cfg: RunWebpackCfg) {
   const { cwd, args = [], env = process.env, devServer = false, prefix } = cfg
+  console.log(cwd);
   const dfd = pDefer()
   const spawned = spawn(
-    devServer
+  `bash.exe ${devServer
```

Comment:
this is unnecessary, one can simply add the `.cmd` at the end to get the windows version